### DISCUSSION TITLE
Font size increase option in IntelliJ plugin

### DISCUSSION THREAD

**Discussion by subin-dti** [community/community#112462] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

It would be helpful if there is an option to increase the font size in the copilot for intellij plugin.

**Comment by ghostinhershell** [comment]
Hi @subin-dti, 

Thanks so much for reaching out in the community. While researching your question I did find this resource that may help answer your question: https://www.jetbrains.com/help/idea/settings-editor-font.html. Please take a look and let me know if you have any additional quesitons!

**Reply by laszlo-bw** [reply]
Workaround requires changing the editor font size, which I certainly don't want to do (dunno about the OP).

Is there any new word on how the github copilot text can be separately resized in intellij?

**Reply by marcelbrode** [reply]
For me, that didn't even work with the GitHub-plugin. MR text are as small as before.

**Reply by liang12431** [reply]
https://youtrack.jetbrains.com/articles/SUPPORT-A-524/How-to-adjust-AI-Assistant-Chat-font-size  This should help you.

**Comment by yuval-ap** [comment]
Still an issue, Same as @marcelbrode changing editor font size didnt help. 




PyCharm 2025.1 (Community Edition)
Build #PC-251.23774.444, built on April 15, 2025
Source revision: 75830c4c8b807
Runtime version: 21.0.6+9-b895.109 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.awt.X11.XToolkit
Linux 6.8.0-58-generic
Ubuntu 24.04.2 LTS; glibc: 2.39
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 2048M
Cores: 8
Registry:
  ide.experimental.ui=true
  llm.show.ai.promotion.window.on.start=false
Non-Bundled Plugins:
  com.github.copilot (1.5.42-243)
Current Desktop: ubuntu:GNOME

**Comment by yuval-ap** [comment]
not really, the drop down menu looks very different in PyCharm 2025.2. In
particular, no "font size" option.

--
Thanks, Yuval.
